dotnet-desktop-guide/net/winforms/controls-design/designer-properties-overview.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Design-time properties overview
+description: Learn about how the Windows Forms Designer interacts with control and form properties during design-time.
+ms.date: 07/14/2023
+ms.topic: overview
+no-loc: ["UserControl"]
+dev_langs:
+- "csharp"
+- "vb"
+f1_keywords: 
+- "UserControl"
+helpviewer_keywords: 
+- "controls [Windows Forms], composite"
+---
+
+# Design-time properties for custom controls (Windows Forms .NET)
+
+This article teaches you about how properties are handled for controls in the Windows Forms Visual Designer in Visual Studio.
+
+Every control inherits many properties from the base class <xref:System.Windows.Forms.Control?displayProperty=nameWithType>, such as:
+
+- <xref:System.Windows.Forms.Control.Enabled%2A>
+- <xref:System.Windows.Forms.Control.Font%2A>
+- <xref:System.Windows.Forms.Control.ForeColor%2A>
+- <xref:System.Windows.Forms.Control.Focused%2A>
+- <xref:System.Windows.Forms.Control.Visible%2A>
+- <xref:System.Windows.Forms.Control.Width%2A>
+
+When creating a control, you can define new properties and control how they appear in the designer.
+
+[!INCLUDE [desktop guide under construction](../../includes/desktop-guide-preview-note.md)]
+
+## Define a property
+
+Any public property with a **get** accessor defined by a control is automatically visible in the Visual Studio **Properties** window. If the property also defines a **set** accessor, the property can be changed in the **Properties** window. However, properties can be explicitly displayed or hidden from the **Properties** window by applying the <xref:System.ComponentModel.BrowsableAttribute>. This attribute takes a single boolean parameter to indicate whether or not it's displayed. For more information about attributes, see [Attributes (C#)](/dotnet/csharp/programming-guide/concepts/attributes/index) or [Attributes overview (Visual Basic)](/dotnet/visual-basic/programming-guide/concepts/attributes/index).
```

Comment:
We should add a comment, that complex properties might need a type converter, if they cannot implicitly be converted from `string` to its native type and vice versa?